### DISCUSSION TITLE
Cancelación curso

### DISCUSSION THREAD

**Discussion by juliancorredor99** [rramosp/ai4eng.v1#138] [00 - General]
Buenos días.

Lo que pasa es que yo matriculé el curso por interés propio, para aprender y también por el certificado, pero cuando matriculé me di cuenta que no es necesario tener el curso matriculado para obtener el certificado, entonces si cancelo la materia puedo seguir haciendo el curso y obtener el certificado al final? muchas gracias.

**Comment by rramosp** [comment]
te tienes que registrar en el curso abierto para eso, no en el de la udea.
en el de la udea, además del certificado quedan los créditos y la materia
correspondiente en tu programa

On Mon, Jun 13, 2022 at 10:04 AM juliancorredor99 ***@***.***>
wrote:

> Buenos días.
>
> Lo que pasa es que yo matriculé el curso por interés propio, para aprender
> y también por el certificado, pero cuando matriculé me di cuenta que no es
> necesario tener el curso matriculado para obtener el certificado, entonces
> si cancelo la materia puedo seguir haciendo el curso y obtener el
> certificado al final? muchas gracias.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/138>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIJ6FQ7ESSJGB3NLXDLVO5EX3ANCNFSM5YURHULA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.

**Reply by juliancorredor99** [reply]
profe, lo que pasa es que estoy en ultimo semestre y como te comento pedí la materia para hacer el curso y obtener el certificado, no porque necesite los créditos o la aprobación de esta.

entonces, quiero saber como procedo ya que creo, que estoy registrado pero por la udea, más no por el curso abierto.
mandaría de nuevo la inscripción pero por otro correo, por ejemplo mi correo personal, mientras que por el de la udea hago la cancelación del curso normal?

**Comment by rramosp** [comment]
sí, registrate por el curso abierto y luego te elimino de la sesión de la
udea

On Mon, Jun 13, 2022 at 11:53 AM juliancorredor99 ***@***.***>
wrote:

> profe, lo que pasa es que estoy en ultimo semestre y como te comento pedí
> la materia para hacer el curso y obtener el certificado, no porque necesite
> los créditos o la aprobación de esta.
>
> entonces, quiero saber como procedo ya que creo, que estoy registrado pero
> por la udea, más no por el curso abierto.
> mandaría de nuevo la inscripción pero por otro correo, por ejemplo mi
> correo personal, mientras que por el de la udea hago la cancelación del
> curso normal?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/138#discussioncomment-2940353>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIK4NZHMJD55D35IQUTVO5RSNANCNFSM5YURHULA>
> .
> You are receiving this because you commented.Message ID: <rramosp/ai4eng.
> ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.

**Reply by juliancorredor99** [reply]
listo profe, ya me me registré por el curso abierto, lo hice con el mismo correo de la universidad, ya que con este correo es que tengo vinculad mi cuenta de github, muchas gracias

**Reply by juliancorredor99** [reply]
profe, aún sigo registrado en el curso de la udea, por lo cual me aparecen todavia dos cursos, supongo que uno es de la UdeA y el otro es el curso abierto, por favor me podrías quitar el de la udea para empezar a hacer los laboratorios, muchas gracias.

**Comment by rramosp** [comment]
cómo es tu correo??

On Wed, Jun 29, 2022 at 11:22 AM juliancorredor99 ***@***.***>
wrote:

> profe, aún sigo registrado en el curso de la udea, por lo cual me aparecen
> todavia dos cursos, supongo que uno es de la UdeA y el otro es el curso
> abierto, por favor me podrías quitar el de la udea para empezar a hacer los
> laboratorios, muchas gracias.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/138#discussioncomment-3049876>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIOFKCHN2JWKJQBVLX3VRRZ5XANCNFSM5YURHULA>
> .
> You are receiving this because you commented.Message ID: <rramosp/ai4eng.
> ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.

**Reply by juliancorredor99** [reply]
julian.corredor@udea.edu.co

**Comment by rramosp** [comment]
ya lo tienes

On Wed, Jun 29, 2022 at 11:22 AM juliancorredor99 ***@***.***>
wrote:

> profe, aún sigo registrado en el curso de la udea, por lo cual me aparecen
> todavia dos cursos, supongo que uno es de la UdeA y el otro es el curso
> abierto, por favor me podrías quitar el de la udea para empezar a hacer los
> laboratorios, muchas gracias.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/138#discussioncomment-3049876>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIOFKCHN2JWKJQBVLX3VRRZ5XANCNFSM5YURHULA>
> .
> You are receiving this because you commented.Message ID: <rramosp/ai4eng.
> ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.
